.docker/opt/configure-aiida.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash -e
+
+# This script is executed whenever the docker container is (re)started.
+
+#===============================================================================
+# debugging
+set -x
+
+#===============================================================================
+# environment
+reentry scan
+
+export SHELL=/bin/bash
+
+#===============================================================================
+# setup AiiDA
+aiida_backend=django
+
+if [ ! -d /home/$SYSTEM_USER/.aiida ]; then
+    verdi setup                                \
+        --profile default                      \
+        --non-interactive                      \
+        --email some.body@xyz.com              \
+        --first-name Some                      \
+        --last-name Body                       \
+        --institution XYZ                      \
+        --db-backend $aiida_backend            \
+        --db-username aiida                    \
+        --db-password aiida_db_passwd          \
+        --db-name aiidadb                      \
+        --db-host localhost                    \
+        --db-port 5432                         \
+        --repository /home/$SYSTEM_USER/aiida_repository
+
+   echo 'eval "$(verdi completioncommand)"' >> /home/$SYSTEM_USER/.bashrc
+
+   verdi profile setdefault default
+fi
+
+#===============================================================================
+# perform the database migration if needed
+#
+verdi daemon start || ( verdi daemon stop && verdi database migrate --force && verdi daemon start)
```

Comment:
I think that the failing of `verdi daemon start` is not the best way of determining whether one needs to migrate the database. Maybe it is best to just run `verdi database migrate -f` before it, since if the database is up to date it is a no-op. If we really wanted, we could make a `verdi database migration-status` command that will fail if it needs to be updated and if it is up to date will return 0.

Reply:
ok

Reply:
I don't really need `verdi database migration-status` I think just run `verdi database migrate -f` is fine for my use case.

However, it would be nice to have possibility to check for an existing profile. It can also be integrated into `verdi profile list`. Something like `verdi profile list <profile name>`. Currently it doesn't work

Reply:
Currently I do `verdi profile show <profile name> &> /dev/null`

Reply:
How about `verdi profile show <profile name> || ...`
(or `if verdi profile show <sdsda> then ...`)?